sqlx-core/src/sqlite/transaction.rs

```diff
@@ -26,3 +30,54 @@ impl TransactionManager for SqliteTransactionManager {
         conn.worker.start_rollback().ok();
     }
 }
+
+/// Transaction initiation options for SQLite.
+#[derive(Clone, Copy, Debug, Default, PartialEq, Eq)]
+pub struct SqliteTransactionOptions {
+    pub(crate) behavior: SqliteTransactionBehavior,
+}
+
+impl SqliteTransactionOptions {
+    pub fn behavior(mut self, behavior: SqliteTransactionBehavior) -> Self {
+        self.behavior = behavior;
+        self
+    }
+}
+
+impl From<SqliteTransactionBehavior> for SqliteTransactionOptions {
+    fn from(behavior: SqliteTransactionBehavior) -> Self {
+        Self { behavior }
+    }
+}
+
+/// Transaction behaviors for SQLite.
```

Comment:
Might be worth linking to https://sqlite.org/lang_transaction.html and/or https://www.sqlite.org/isolation.html